test/tsp.test.js

```diff
@@ -43,7 +43,7 @@ tap.test('Test TSP', function(assert) {
   TSP.Solve(searchOpts, function (err, solution) {
     assert.ifError(err, 'Solution can be found');
 
-    assert.type(solution, Array, 'Solution is Array of locations');
+    assert.ok(solution instanceof Array, 'Solution is Array of locations');
```

Comment:
Why?

Reply:
This one is not related to upgrading to use latest node-cpp-skel, so I'll let @danpat answer.

Reply:
It's only semi-related.  `node-cpp-skel` suggests using `tape`, but this project was using `tap` for testing - the change above is because `tape` doesn't support the `.type()` test function.